### ISSUE TITLE
Button icons don't show in safari and home assistant companion app on iPhone 

### ISSUE THREAD

**Issue by loopphoto** [marrobHD/tv-card#26]
Thanks for the great work. I've been using tv-card for a while and it's been perfect until recently(I don't remember when). Now none of the button icons show in safari or on the home assistant companion app on iOS. Works fine on Chrome(tested on Mac
<img width="1073" alt="Screenshot 2022-01-19 at 22 46 36" src="https://user-images.githubusercontent.com/30428657/150211152-ba0a9852-19ec-4513-a8bf-c83db3e3b55d.png">

**Comment by marrobHD** [comment]
This could be related to Apples WebKit changes. I don't have any apple devices/a clue how to fix this.

**Comment by guilhermecespedes** [comment]
i am causing the same problem with my android device.

**Comment by marrobHD** [comment]
> i am causing the same problem with my android device.

I can't verify this. Could you please give me your card configuration?

**Comment by guilhermecespedes** [comment]
My code is configured as follows.

#
```
  - type: 'custom:tv-card'
    entity: remote.rm4_sala_remote
    tv: true
    power_on:
      service: remote.send_command
      service_data:
        entity_id: remote.rm4_sala_remote
        device: tv_sala
        command: power_on
    power_off:
      service: remote.send_command
      service_data:
        entity_id: remote.rm4_sala_remote
        device: tv_sala
        command: power_off
    back:
      service: remote.send_command
      service_data:
        entity_id: remote.rm4_sala_remote
        device: tv_sala
        command: voltar
    source:
      service: remote.send_command
      service_data:
        entity_id: remote.rm4_sala_remote
        device: tv_sala
        command: source
    home:
      service: remote.send_command
      service_data:
        entity_id: remote.rm4_sala_remote
        device: tv_sala
        command: home
    channelup:
      service: remote.send_command
      service_data:
        entity_id: remote.rm4_sala_remote
        device: tv_sala
        command: canal_cima
    info:
      service: media_player.play_media
      service_data:
        entity_id: media_player.tv_quarto
        media_content_id: KEY_INFO
        media_content_type: send_key
    channeldown:
      service: remote.send_command
      service_data:
        entity_id: remote.rm4_sala_remote
        device: tv_sala
        command: canal_baixo
    up:
      service: remote.send_command
      service_data:
        entity_id: remote.rm4_sala_remote
        device: tv_sala
        command: cima
    left:
      service: remote.send_command
      service_data:
        entity_id: remote.rm4_sala_remote
        device: tv_sala
        command: esquerda
    select:
      service: remote.send_command
      service_data:
        entity_id: remote.rm4_sala_remote
        device: tv_sala
        command: ok
    right:
      service: remote.send_command
      service_data:
        entity_id: remote.rm4_sala_remote
        device: tv_sala
        command: direita
    down:
      service: remote.send_command
      service_data:
        entity_id: remote.rm4_sala_remote
        device: tv_sala
        command: baixo
    volume_mute:
      service: remote.send_command
      service_data:
        entity_id: remote.rm4_sala_remote
        device: tv_sala
        command: mudo
    volume_up:
      service: remote.send_command
      service_data:
        entity_id: remote.rm4_sala_remote
        device: tv_sala
        command: volume_aumentar
    volume_down:
      service: remote.send_command
      service_data:
        entity_id: remote.rm4_sala_remote
        device: tv_sala
        command: volume_diminuir
```




I apologize, but I don't speak English and I'm doing the translations via google translator.

**Comment by marrobHD** [comment]
It's quite annoying to remove the quotes in your yaml snippet. So please check this guide out: [click me](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code-and-syntax-highlighting)

Your configuration looks fine so far...

**Comment by guilhermecespedes** [comment]
```
  - type: 'custom:tv-card'
    entity: remote.rm4_sala_remote
    tv: true
    power_on:
      service: remote.send_command
      service_data:
        entity_id: remote.rm4_sala_remote
        device: tv_sala
        command: power_on
    power_off:
      service: remote.send_command
      service_data:
        entity_id: remote.rm4_sala_remote
        device: tv_sala
        command: power_off
    back:
      service: remote.send_command
      service_data:
        entity_id: remote.rm4_sala_remote
        device: tv_sala
        command: voltar
    source:
      service: remote.send_command
      service_data:
        entity_id: remote.rm4_sala_remote
        device: tv_sala
        command: source
    home:
      service: remote.send_command
      service_data:
        entity_id: remote.rm4_sala_remote
        device: tv_sala
        command: home
    channelup:
      service: remote.send_command
      service_data:
        entity_id: remote.rm4_sala_remote
        device: tv_sala
        command: canal_cima
    info:
      service: media_player.play_media
      service_data:
        entity_id: media_player.tv_quarto
        media_content_id: KEY_INFO
        media_content_type: send_key
    channeldown:
      service: remote.send_command
      service_data:
        entity_id: remote.rm4_sala_remote
        device: tv_sala
        command: canal_baixo
    up:
      service: remote.send_command
      service_data:
        entity_id: remote.rm4_sala_remote
        device: tv_sala
        command: cima
    left:
      service: remote.send_command
      service_data:
        entity_id: remote.rm4_sala_remote
        device: tv_sala
        command: esquerda
    select:
      service: remote.send_command
      service_data:
        entity_id: remote.rm4_sala_remote
        device: tv_sala
        command: ok
    right:
      service: remote.send_command
      service_data:
        entity_id: remote.rm4_sala_remote
        device: tv_sala
        command: direita
    down:
      service: remote.send_command
      service_data:
        entity_id: remote.rm4_sala_remote
        device: tv_sala
        command: baixo
    volume_mute:
      service: remote.send_command
      service_data:
        entity_id: remote.rm4_sala_remote
        device: tv_sala
        command: mudo
    volume_up:
      service: remote.send_command
      service_data:
        entity_id: remote.rm4_sala_remote
        device: tv_sala
        command: volume_aumentar
    volume_down:
      service: remote.send_command
      service_data:
        entity_id: remote.rm4_sala_remote
        device: tv_sala
        command: volume_diminuir
```

**Comment by marrobHD** [comment]
Works for me. Can you check out if the buttons are still there but the icons are not displayed? Just click somewhere in the card without buttons.

**Comment by guilhermecespedes** [comment]
buttons work, only icons are not displayed. No computer works normally only on android app that doesn't.

https://user-images.githubusercontent.com/85122323/153772148-389740a0-67f9-4341-9f17-71f1f8d34ff0.mp4

Chrome Windowns 10:
![image](https://user-images.githubusercontent.com/85122323/153772213-248483ba-2e0d-4a25-8cba-d9c8e98e66ff.png)

**Comment by marrobHD** [comment]
Strange behaviour. For me and my other Android devices it's working. Maybe try to redownload the card

**Comment by guilhermecespedes** [comment]
it's pretty weird! I already tried to reinstall but unfortunately it didn't work. I thought it might be the theme I'm using, but with the default theme it doesn't show the icons either.

I tested it on other android devices, but the same thing happens! By browser (Chrome) on android phone displays the icons normally. It just doesn't work in the HA app.

I will keep trying to find the solution.
